include/net/buf.h

```diff
@@ -993,6 +992,9 @@ struct net_buf_pool {
 	/** Number of uninitialized buffers */
 	uint16_t uninit_count;
 
+	/* Size of user data allocated to this pool */
+	uint8_t user_data_len;
```

Comment:
Same here. Should be `user_data_size`. It's also the same what we used to have before #5721 (except that it was `uint16_t` back then)